about.html

```diff
@@ -12,57 +12,66 @@
 </head>
 
 <body>
+  <!-- sree code -->
+  <header>
+    <nav>
+        <img src="img/lambda-black.png"/>
+        <a class="nav1" href="index.html" >Home</a>
+        <a class="nav1" href="about.html" >About</a>
+        <a class="nav1" href="#" >Products</a>
+        <a class="nav1" href="#" >Blog</a>
+        <a class="nav1" href="#" >Contact</a>
+    </nav>
+</header>
+<div class="jumbo">
+    <img  src="img/jumbo-about.png"/>
+</div>
+  <!-- sree code -->
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+     <div class="about-div">
+      <h2>About</h2>
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      <div class="line"></div>
+    </div>
+
+    <section class="about-details">    
+      <div class="strategy-div">
+        <img src="img/about-plan.png" alt="strategy">    
+        <h2>Strategy</h2>   
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <button class="btn-learm-more">Learn More</button>
+        <div class="line"></div>
+    </div>
```

Comment:
One thing you can do here is reuse the class.  Each of these divs we-work-div, strategy-div places-div, etc. use the exact same CSS so you could create one class and reuse it.